app/api/__init__.py

```diff
@@ -28,12 +29,14 @@
 api.route(EventList, 'event_list', '/events')
 api.route(EventDetail, 'event_detail', '/events/<int:id>', '/tickets/<int:ticket_id>/event',
           '/microlocations/<int:microlocation_id>/event', '/social_links/<int:social_link_id>/event',
-          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event', '/call_for_papers/<int:call_for_paper_id>/event')
+          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event',
+          '/event_copyright/<int:copyright_id>/event', '/call_for_papers/<int:call_for_paper_id>/event')
```

Comment:
> /session_types shouldn't exist

@niranjan94  never said /session_types/<int:sessiontype_id> shouldn't exist. Infact what I discussed with him, he said patch, delete should be functional on a link like /session_types/<int:sessiontype_id> . @niranjan94 your comments?